ui/app/AppLayouts/Wallet/HistoryTab.qml

```diff
@@ -1,8 +1,140 @@
 import QtQuick 2.3
+import "../../../imports"
 
 Item {
-    Text {
-        id: name3
-        text: "HISTORY"
+    Component {
+      id: transactionListItem
+
+      Item {
+        anchors.right: parent.right
+        anchors.rightMargin: 0
+        anchors.left: parent.left
+        anchors.leftMargin: 0
+        height: 64
+
+        Item {
+
+          Rectangle {
+            id: assetIcon
+            color: "gray"
+            width: 40
+            height: 40
+            anchors.left: parent.left
+            anchors.leftMargin: 0
+            anchors.top: parent.top
+            anchors.topMargin: 12
+            radius: 50
+          }
```

Comment:
Went with a gray circle for now as I'm not sure how we get the asset/currency icon from a transaction. I guess we need to map them somehow to their respective contracts?

Reply:
Yeah, we need a way to transfer the address to the token symbol so that we can then use right image from our token svgs.
I'm not sure if right now we have a constants file with an object of that or if maybe there is a way to get the symbol from the network